aider.el

```diff
@@ -142,20 +142,19 @@ Affects the system message too.")
 ;; Removed the default key binding
 ;; (global-set-key (kbd "C-c a") 'aider-transient-menu)
 
-(defun aider-buffer-name-from-git-repo-path (git-repo-path home-path)
-  "Generate the Aider buffer name based on the GIT-REPO-PATH and HOME-PATH.
+(defun aider-buffer-name-from-git-repo-path (git-repo-path)
+  "Generate the Aider buffer name based on the GIT-REPO-PATH.
 If not in a git repository, an error is raised."
-  (let* ((relative-path (substring git-repo-path (length home-path))))
-    (format "*aider:%s*" (concat "~" (replace-regexp-in-string "\n" "" relative-path)))))
+  (let ((hash8 (substring (secure-hash 'md5 git-repo-path) 0 8)))
+    (format "*aider:~%s%s*" git-repo-path hash8)))
```

Comment:
not sure if we need the hash8 as part of the buffer name, cause it might not provide additional information